examples/all-clusters-app/esp32/main/CHIPDeviceManager.cpp

```diff
@@ -99,6 +99,13 @@ CHIP_ERROR CHIPDeviceManager::Init(CHIPDeviceManagerCallbacks * cb)
 void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath & path, uint8_t mask, uint8_t type, uint16_t size,
                                        uint8_t * value)
 {
+    TaskHandle_t task = xTaskGetCurrentTaskHandle();
+    const char * name = pcTaskGetName(task);
+    if (!strcmp(name, "CHIP"))
```

Comment:
Note that this test is backwards; you may want to cherry-pick the fix for that too.